openedx/features/caliper_tracking/tests/expected/edx.bookmark.added.json

```diff
@@ -0,0 +1,39 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id": "urn:uuid:d4618c23-d612-4709-8d9a-478d87808067",
+  "type": "AnnotationEvent",
+  "actor": {
+    "id": "http://localhost:18000/u/ali",
+    "name": "ali",
+    "type": "Person"
+  },
+  "action": "Bookmarked",
+  "object": {
+    "id": "http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course/courseware/d8a6192ade314473a78242dfeedfbf5b/edx_introduction/1?activate_block_id=block-v1%3AedX%2BDemoX%2BDemo_Course%2Btype%40vertical%2Bblock%40vertical_0270f6de40fc",
+    "type": "Page"
```

Comment:
Move the key/value pairs `course_id`, ` bookmark_id` and `component_usage_id` to the `extensions` key inside `object`

Reply:
done.
I have also moved the component_type also as it is also related to object.